lib/backend/syncersv1/updateprocessors/rules_test.go

```diff
@@ -60,10 +59,6 @@ var _ = Describe("Test the Rules Conversion Functions", func() {
 				NotNets:     []string{"192.168.40.1/32"},
 				NotSelector: "has(label1)",
 				NotPorts:    []numorstring.Port{port443},
-				ServiceAccounts: &apiv3.ServiceAccountMatch{
```

Comment:
Why'd you drop this bit (and the part below)?

Reply:
The service accounts pieces affected the selectors now but did not add much to this specific test cases here. Hence I took them out of here.
The impact of service accounts on the selector is explicitly captured in the UT [here](https://github.com/colabsaumoh/libcalico-go/blob/0aecad65e5dc40bb786d4ea3668f8491e8e6683e/lib/backend/syncersv1/updateprocessors/rules_test.go#L372) and [here](https://github.com/colabsaumoh/libcalico-go/blob/0aecad65e5dc40bb786d4ea3668f8491e8e6683e/lib/backend/syncersv1/updateprocessors/rules_test.go#L445)